04_initial_data_prep.R

```diff
@@ -87,7 +87,12 @@ if("ProjectTimeID" %in% colnames(ProjectsInHMIS)){
 
 # This dataset is used when we need an unduplicated concise df for project
 Project0 <<- Project %>% 
-  select(ProjectID, ProjectName, OrganizationID, OrganizationName, ProjectType) %>%
+  select(ProjectID,
+         ProjectName,
+         OrganizationID,
+         OrganizationName,
```

Comment:
If ProjectStaging merges OrganizationName into the Project dataset, then how do we have OrganizationName when we create Project0 here? I'm guessing we should be using ProjectStaging here, instead of Project. But since the app hasn't thrown an error about it (?) maybe Project already has OrganizationName? In which case, we don't need ProjectStaging...